torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10664,6 +10683,16 @@ def error_inputs_mean(op_info, device, **kwargs):
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
            sample_inputs_func=sample_inputs_broadcast_to),
+    OpInfo('broadcast_shapes',
+           op=torch.broadcast_shapes,
+           ref=np.broadcast_shapes,
+           dtypes=_dispatch_dtypes((torch.bool,)),
+           dtypesIfCUDA=_dispatch_dtypes(),  # broadcast_shapes is device independent
```

Comment:
Modeling dtype-agnostic functions is not something our system is really designed for, but maybe we should fake it by pretending this op supports just one dtype -- float32 -- and make a note that we're doing that so it just generates a single float32 variant for each test?